demux/demux_mkv.c

```diff
@@ -1862,6 +1869,45 @@ static int demux_mkv_open_sub(demuxer_t *demuxer, mkv_track_t *track)
     sh->codec->extradata = track->private_data;
     sh->codec->extradata_size = track->private_size;
 
+    if (!strcmp(sh->codec->codec, "arib_caption")) {
+        struct AVCodecParameters **lav = talloc_ptrtype(track, lav);
+        if (!lav)
```

Comment:
`talloc_ptrtype -> ta_xnew_ptrtype -> ta_oom_g(ta_new_ptrtype(__VA_ARGS__)) `
so I think unless the ta_ stuff somehow gets overridden by something this check should never fail.